instrumentation/logback-classic-1.2/src/main/java/com/nr/agent/instrumentation/logbackclassic12/AgentUtil.java

```diff
@@ -58,6 +62,13 @@ public static void recordNewRelicLogEvent(String message, long timeStampMillis,
                 logEventMap.put(LOG_LEVEL, level);
             }
 
+            if (t != null && isApplicationForwardStackTracesEnabled()) {
```

Comment:
My co-worker and expected this to be the default behavior, we'd be in-favor of this not even being behind a flag.
I assume this was omitted to save data egress / ingress.